benchmarks/helics/EchoMessage.hpp

```diff
@@ -9,72 +9,71 @@ SPDX-License-Identifier: BSD-3-Clause
 #include "helics/application_api/MessageFederate.hpp"
 #include "helics/core/ActionMessage.hpp"
 
-
-using helics::operator"" _t ;
+using helics::operator"" _t;
 // static constexpr helics::Time tend = 3600.0_t;  // simulation end time
 /** class implementing the hub for an echo test*/
 class EchoMessageHub
 {
   public:
-    helics::Time finalTime = helics::Time (100, time_units::ms);  // final time
+    helics::Time finalTime = helics::Time(100, time_units::ms);  // final time
   private:
     std::unique_ptr<helics::MessageFederate> mFed;
     helics::Endpoint ept;
     bool initialized = false;
-    bool readyToRun = false;
+    bool readyToRun  = false;
 
   public:
-    EchoMessageHub () = default;
+    EchoMessageHub() = default;
 
-    void run (std::function<void()> callOnReady = {})
+    void run(std::function<void()> callOnReady = {})
     {
         if (!readyToRun)
         {
-            makeReady ();
+            makeReady();
         }
         if (callOnReady)
         {
-            callOnReady ();
+            callOnReady();
         }
-        mainLoop ();
+        mainLoop();
     };
 
-    void initialize (const std::string &coreName)
+    void initialize(const std::string & coreName)
     {
         std::string name = "echohub";
         helics::FederateInfo fi;
         fi.coreName = coreName;
-        mFed = std::make_unique<helics::MessageFederate> (name, fi);
-        ept = mFed->registerGlobalEndpoint ("echo");
+        mFed        = std::make_unique<helics::MessageFederate>(name, fi);
```

Comment:
Do we like aligning the '='?

Reply:
I think that while it might look sort of nice, it is annoying trying to align the `=` and is likely to guarantee that any changes made will require running clang-format.

Reply:
I think aligning `=` creates unnecessary diffs, so I'd be okay with not adding this option, even though it can be easier to read sometimes.